straxen/scada.py

```diff
@@ -22,12 +24,15 @@ def __init__(self, context=None):
             if you would like to query data via run_ids.
         """
         try:
-            self.SCData_URL = straxen.get_secret('SCData_URL')
-            self.SCLastValue_URL = straxen.get_secret('SCLastValue_URL')
-            self.SCADA_SECRETS = straxen.get_secret('SCADA_SECRETS')
+            self.SCData_URL = uconfig.get('scada', 'scdata_url')
+            self.SCLastValue_URL = uconfig.get('scada', 'sclastvalue_url')
+            self.SCADA_SECRETS = dict(QueryType=uconfig.get('scada', 'querytype'),
+                                      username=uconfig.get('scada', 'username'),
+                                      api_key=uconfig.get('scada', 'api_key')
+                                      )
         except ValueError:
-            raise ValueError('Cannot load SCADA information, from xenon'
-                             ' secrets. SCADAInterface cannot be used.')
+            raise ValueError(f'Cannot load SCADA information, from xenon'
+                             ' config at {uconfig.config_path}. SCADAInterface cannot be used.')
```

Comment:
If uconfig is None, this will become quite an unintuitive traceback, this is very well possible. Please keep this case in mind.

Reply:
Good point, thanks @jorana. Will fix.

Reply:
Fixed now